dmsite/venv/src/Authentication/NewAccountTextField.js

```diff
@@ -0,0 +1,186 @@
+/**
+ * Created by christinadaley on 1/21/19.
+ */
+
+import React from "react";
+import { render, ReactDOM } from "react-dom";
+import { TransitionMotion, spring } from 'react-motion';
+import { Component } from 'reactstrap';
+import { withRouter } from "react-router-dom";
+import {
+    Form, Input, Tooltip, Icon, Cascader, Select, Row, Col, Checkbox, Button, AutoComplete,
+} from 'antd';
+
+const { Option } = Select;
+const AutoCompleteOption = AutoComplete.Option;
+
+
+class NewAccountTextField extends React.Component {
+    state = {
+        confirmDirty: false,
+        autoCompleteResult: [],
+    };
+
+    handleSubmit = (e) => {
+        e.preventDefault();
+        this.props.form.validateFieldsAndScroll((err, values) => {
+            if (!err) {
+                console.log('Received values of form: ', values);
+            }
+        });
+    }
+
+    handleConfirmBlur = (e) => {
+        const value = e.target.value;
+        this.setState({ confirmDirty: this.state.confirmDirty || !!value });
+    }
+
+    compareToFirstPassword = (rule, value, callback) => {
+        const form = this.props.form;
+        if (value && value !== form.getFieldValue('password')) {
+            callback('Two passwords that you enter is inconsistent!');
+        } else {
+            callback();
+        }
+    }
+
+    validateToNextPassword = (rule, value, callback) => {
+        const form = this.props.form;
+        if (value && this.state.confirmDirty) {
+            form.validateFields(['confirm'], { force: true });
+        }
+        callback();
+    }
+
+    handleWebsiteChange = (value) => {
+        let autoCompleteResult;
+        if (!value) {
+            autoCompleteResult = [];
+        } else {
+            autoCompleteResult = ['.com', '.org', '.net'].map(domain => `${value}${domain}`);
+        }
+        this.setState({ autoCompleteResult });
+    }
+
+    render() {
+        const { getFieldDecorator } = this.props.form;
+        const { autoCompleteResult } = this.state;
+
+        const formItemLayout = {
+            labelCol: {
+                xs: { span: 24 },
```

Comment:
Just wondering why do we need to differentiate xs vs s?

Reply:
They weren't needed! I took them out! Thank you!